src/App.js

```diff
@@ -1,29 +1,25 @@
-import logo from './logo.svg';
+/* eslint-disable jsx-quotes */
+/* eslint-disable import/extensions */
+
+import React from 'react';
 import './App.css';
+import Calculator from './components/Calculator';
+
```

Comment:
- [ ] Let's try not to disable Linter checks as they help us write better and more professional code. Disabling Linters defies its purpose. Please fix the error 👍🏽
- [ ] The import for the Calculator component from the components directory is in uppercase but the file is actually in lowercase and for that reason, the wrong import is being done and the app is not starting or loading that page. Kindly work on that 👍🏽